phpseclib/Crypt/Rijndael.php

```diff
@@ -755,7 +755,7 @@ function _subWord($word)
     {
         static $sbox;
         if (empty($sbox)) {
-            list(,,,, $sbox) = $this->_getTables();
+            list(, , , , $sbox) = $this->_getTables();
```

Comment:
In particular, I'd argue that this call is not empty because it contains the `$sbox` variable. Not 100% sure. Maybe the manual/documentation should be a bit more clear.